dts/arm/nxp/nxp_rt118x_cm33.dtsi

```diff
@@ -27,6 +33,10 @@
 		flexspi1: spi@525e0000 {
 			reg = <0x525e0000 0x4000>,<0x38000000 DT_SIZE_M(128)>;
 		};
+
```

Comment:
There are changes in this commit and file that do not appear to be related to the the hyperram support.